loadtests/Makefile

```diff
@@ -12,7 +12,7 @@ INSTALL = ARCHFLAGS=$(ARCHFLAGS) ./venv/bin/pip install
 .env.install:
 	virtualenv ./venv
 	$(INSTALL) pexpect >> loadtest.log && \
-	$(INSTALL) configparser==3.3.0r2 >> loadtest.log && \
+	$(INSTALL) configparser==3.3.0post2 >> loadtest.log && \
```

Comment:
3.3.0post2 is working at least according to travis ;) I'm doing another run without the `python` language, just to see if it succeeds to, as you advised ;)